applications/new_schema/types.py

```diff
@@ -87,3 +93,62 @@ def resolve_harbor_choices(self, info, **kwargs):
     @view_permission_required(BerthApplication, BerthLease, CustomerProfile)
     def get_node(cls, info, id):
         return super().get_node(info, id)
+
+
+WinterStorageMethodEnum = graphene_enum(WinterStorageMethod)
+
+
+class WinterStorageAreaChoiceType(DjangoObjectType):
+    winter_storage_area = graphene.String()
```

Comment:
why isn't this `required=True`? 🤔

Reply:
this maps to `winter_storage_area.servicemap_id` (like in `HarborChoiceType -> harbor`), but the field is nullable which causes api call to fail in case of nulls.

Reply:
ah, yes, true, WS areas do not have servicemap ids. this is ok then, we will change this later, if needed.